### ISSUE TITLE
Occasional Blank left-side drawer controller

### ISSUE THREAD

**Issue by shiyunhe** [mutualmobile/MMDrawerController#30]
Occasionally, the left side drawer controller can be blank.

![img_2850](https://f.cloud.github.com/assets/4458320/564970/2c7f7042-c5c5-11e2-948a-8f6f35cc1e18.png)
![img_2849](https://f.cloud.github.com/assets/4458320/564971/3a224184-c5c5-11e2-8480-f2cdcbb3a0b4.png)

**Comment by lionel-SF** [comment]
Hello, friends

i meet this and reproduce it.
At LeftSideViewController we set difference url addresses which is a UITableViewController,
At CenterViewController we set a weView to load the url from leftsideVC; so we can klick left side tableView and change the url address we want.
Important: we want the web view to handle the session cookie for url, so we do not need login again after invalid.After my APP enter background more than 30 minutes, the session cookie is invalid, login again by using TouchID then see the same page, but it failures , and show the TouchID again , and TouchID success then you will login success and find the left side VC black.
More: This only happened at APP install by ipa file style. and we set the session valid time to 1 minute at DEBUG ,we do not see it, so it hard for me to follow debug.
Looking forward, who can help ? or any ideas.
![img_4986](https://cloud.githubusercontent.com/assets/15133909/17992577/3e46fbd8-6b7b-11e6-9193-fa16279cf41d.PNG)
